app/javascript/views/walletServices/braveConnection/NotConnected.tsx

```diff
@@ -59,4 +61,21 @@ const GeminiConnectButton = () => (
   </a>
 );
 
+const BitflyerConnectButton = () => (
+  <a
+    className="btn btn-secondary font-weight-bold mb-2"
+    data-piwik-action="bitflyerConnectClicked"
```

Comment:
looks like all others are capitalized. should this be as well?

Reply:
thanks @maikelmclauflin you're totally right. Fixed